mongodbatlas/processes_test.go

```diff
@@ -52,7 +53,8 @@ func TestProcesses_ListProcesses(t *testing.T) {
 						"replicaSetName": "replica-set-0",
 						"shardName": "shard-0",
 						"typeName": "REPLICA_PRIMARY",
-						"version": "3.6.7"
+						"version": "3.6.7",
+						"userAlias": "zuul",
```

Comment:
[nit] the dangling comma (,) is going to throw serialisation errors, same with line 38